projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx

```diff
@@ -40,6 +41,30 @@ const ConnectedProductCard = ( { admin, slug, children } ) => {
 		navigateToConnectionPage,
 	] );
 
+	/*
+	 * Redirect to connection page if the user is not connected
+	 */
```

Comment:
I vote to remove the installation option if the user isn't connected. Strange experience to be moved for connection after clicking on install, and in the end, it won't be installed. We have a big notice at the top warning the user that isn't connected.

Reply:
Technically it's not impossible to trigger the installation for a disconnected site, but I agree that it may be a sub-optimal experience. I will try removing the option to see how things show up.

Reply:
Removed the connection redirect from the code and added a check for the `error` status, but I think it's not enough yet. I will figure out a way to have the connection state available on the menu to include it on the button condition.

Reply:
Done. I injected the connection state into the product card and use it to decide between showing or not the menu. For me it does make sense to hide the menu everytime there is a connection problem, since it will not show anything that could work disconnected and will not help fix the connection issue, at least for now. Do you agree?